### DISCUSSION TITLE
Install KernelSU Pixel 7 Pro

### DISCUSSION THREAD

**Discussion by enurdiansah** [tiann/KernelSU#1837] [General]
hello please help me to install KernelSU..
I have tried patching & flashing init_boot & boot but KSU is not installed.

Pixel 7 Pro with Project Elixir

**Comment by joyX3** [comment]
Same problem

**Comment by acmor** [comment]
I'm using KernelSU on my Pixel 7 Pro running Lineage 21 since ~1 month. No issues so far.

EDIT: Lineage 21 = Android 14

**Reply by enurdiansah** [reply]
I'm using project elixir, I try patch init_boot but still not working

**Comment by jediadolf** [comment]
What is the android version?

**Reply by enurdiansah** [reply]
A14.. but, i am waiting for A15 release by developer

**Comment by acmor** [comment]
I switched to [KernelSU-Next](https://github.com/rifsxd/KernelSU-Next) on my Pixel 7 Pro and sInce a few weeks I'm running [LineageOS 22.1](https://wiki.lineageos.org/devices/cheetah/) (Android 15).

Here is how I get it up and running:
1. download & install KernelSU app
2. upload the original image: adb push init_boot.img /sdcard/Download/
3. patch the init_boot.img using the KernelSU app
4. download the patched image: adb pull /sdcard/Download/kernelsu_next_patched_20250117_095857.img .
5. reboot to bootloader: adb reboot bootloader
6. flash the patched image: fastboot flash init_boot kernelsu_next_patched_20250117_095857.img
7. reboot the phone: fastboot reboot

**Reply by enurdiansah** [reply]
ok, I will try it later on A15.. because A15 is not released yet

**Reply by elektrinis** [reply]
do you have strong integrity, all banking apps work, including revolut?
